src/dmd/backend/rtlsym.c

```diff
@@ -23,6 +23,10 @@
 static char __file__[] = __FILE__;      /* for tassert.h                */
 #include        "tassert.h"
 
+#if HTOD
+unsigned ALLREGS;
```

Comment:
Not `regm_t` ?

https://github.com/dlang/dmd/blob/4cc487f1edb3ed1e91ec94c98407384d3e372ae8/src/dmd/backend/cgcod.c#L65

Reply:
I think it is because regm_t isn't defined for HTOD. HTOD is the C .h file to D translator, it doesn't generated code.